google-cloud-bigquery/acceptance/bigquery/table_test.rb

```diff
@@ -43,7 +43,11 @@
     if t.nil?
       t = dataset.create_table time_partitioned_table_id do |updater|
         updater.time_partitioning_type = "DAY"
+        updater.time_partitioning_type = "dob"
         updater.time_partitioning_expiration = seven_days
+        updater.schema do |schema|
+          schema.timestamp "dob",   description: "dob description",   mode: :required
+        end
```

Comment:
There's something wrong here, I get:

```sh
  1) Error:
Google::Cloud::Bigquery::Table::bigquery#test_0007_allows tables to be created with time_partioning enabled:
Google::Cloud::InvalidArgumentError: invalid: Invalid value for: dob is not a valid value
    /Users/quartzmo/code/google/codez/google-cloud-ruby/google-cloud-bigquery/lib/google/cloud/bigquery/service.rb:690:in `rescue in execute'
    /Users/quartzmo/code/google/codez/google-cloud-ruby/google-cloud-bigquery/lib/google/cloud/bigquery/service.rb:684:in `execute'
    /Users/quartzmo/code/google/codez/google-cloud-ruby/google-cloud-bigquery/lib/google/cloud/bigquery/service.rb:161:in `insert_table'
    /Users/quartzmo/code/google/codez/google-cloud-ruby/google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb:507:in `create_table'
    /Users/quartzmo/code/google/codez/google-cloud-ruby/google-cloud-bigquery/acceptance/bigquery/table_test.rb:44:in `block (2 levels) in <top (required)>'
```

Reply:
What was the issue here and how did you fix it? After the squash I can't see the update.